src/lib/y2storage/planned_volumes_list.rb

```diff
@@ -271,16 +271,19 @@ def extra_space_candidates
     # Extra space to be assigned to a volume
     #
     # @param volume [PlannedVolume] volume to enlarge
-    # @param available_size [DiskSize] free space to be distributed among
+    # @param total_size [DiskSize] free space to be distributed among
     #    involved volumes
     # @param total_weight [Float] sum of the weights of all involved volumes
+    # @param assigned_size [DiskSize] space already distributed to other volumes
     # @param rounding [DiskSize] size to round up
     #
     # @return [DiskSize]
-    def volume_extra_size(volume, available_size, total_weight, rounding)
-      extra_size = available_size * (volume.weight / total_weight)
+    def volume_extra_size(volume, total_size, total_weight, assigned_size, rounding)
```

Comment:
Also mine (in fact, I was expecting rubocop to complain). But since it's just a private method to do same calculations I decided it was ok in this case.